src/CSSList/Document.php

```diff
@@ -29,7 +29,7 @@ public function __construct($iLineNo = 0)
      *
      * @throws SourceException
      */
-    public static function parse(ParserState $oParserState)
+    public static function parse(ParserState $oParserState): Document
```

Comment:
We also can drop the `@return` PHPDoc annotation here.

Reply:
Fixed it.